src/App.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Greeting from './components/Greeting';
+
+const App = () => (
+  <>
+    <Greeting />
+  </>
```

Comment:
- [ ] As it is in the project requirements, you are required to Create the App component with react-router. This basically means that you should be able to have a "Greeting" route for your project which might also be specified as the root route. For some context, you could consider doing it this way:

```
function App() {
  return (
    <Provider store={store}>
      <BrowserRouter>
        <Routes>
          <Route path="/" element={<Greeting />} />
        </Routes>
      </BrowserRouter>
    </Provider>
  );
}
```

**Note:** Do not forget to install the react router package in your project.